src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -84,39 +87,32 @@ def get(self, request: Request, project: Project, event_id: str) -> Response:
         if event is None:
             raise NotFound(detail="Event not found")
 
-        release_version = event.get_tag("sentry:release")
-
         exception = event.interfaces["exception"].values[int(exception_idx)]
-
-        filename, abs_path = self._get_filename_and_path(exception, frame_idx)
-
-        if not release_version:
-            return self._create_response(issue=SourceMapProcessingIssue.MISSING_RELEASE)
+        raw_stacktrace = exception.raw_stacktrace
+        if raw_stacktrace:
+            # Exception is already source mapped
+            return self._create_response()
 
         try:
-            release = Release.objects.get(
-                organization=project.organization, version=release_version
-            )
-        except Release.DoesNotExist:
+            release = self._extract_release(event, project)
+        except Exception:
```

Comment:
@roggenkemper This could create some bugs where we get some sort of other error but return `MISSING_RELEASE`. It would be better to catch specific errors like `Release.DoesNotExist` (can be multiple error types).